src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -440,9 +440,10 @@ func (s *service) Bootstrapped(ctx thrift.Context) (*rpc.NodeBootstrappedResult_
 // BootstrappedInPlacementOrNoPlacement is designed to be used with cluster
 // management tools like k8s that expected an endpoint that will return
 // success if the node either:
-// 1) Has no cluster placement set yet.
-// 2) Is bootstrapped and durable, meaning it is bootstrapped and is able
-//    to bootstrap the shards it owns from it's own local disk.
+//  1. Has no cluster placement set yet.
+//  2. Is bootstrapped and durable, meaning it is bootstrapped and is able
+//     to bootstrap the shards it owns from it's own local disk.
+//
```

Comment:
this feels like a formatting change, do we need this?

Reply:
Yes I have removed them, thanks for catching these.